terraform/deployments/datagovuk-infrastructure/main.tf

```diff
@@ -45,3 +49,5 @@ provider "helm" {
     }
   }
 }
+
+provider "fastly" { api_key = "test" }
```

Comment:
optional: does it let you set this to something a bit more self-describing, like "not_used"?